hack/jenkins/e2e.sh

```diff
@@ -40,7 +40,13 @@ function join_regex_no_empty() {
 }
 
 # GCP Project to fetch Trusty images.
-TRUSTY_IMAGE_PROJECT="${TRUSTY_IMAGE_PROJECT:-}"
+function get_trusty_image_project() {
+  gsutil cat "gs://trusty-images/image-project.txt"
+  # Clean up gsutil artifacts otherwise the later test stage will complain.
+  rm -rf .config &> /dev/null
+  rm -rf .gsutil &> /dev/null
+}
+TRUSTY_IMAGE_PROJECT="$(get_trusty_image_project)"
```

Comment:
Can this be wrapped in a `if [[ "${JOB_NAME}" =~ -trusty- ]]; then ... fi` block so that we don't need to run it for all the other jobs on this branch? Not a big deal, though.